setup-client.yml

```diff
@@ -91,6 +90,33 @@
       path: /usr/share/khotkeys/spectacle.khotkeys
       state: absent
 
+# Access 2013 ueber Wine einrichten
+  - name: Wineprefix-Tar auf Client kopieren
+    copy:
+      src: /adfc/software/Ansible/wine_access_2013_vorlage.tar.gz
+      dest: /usr/local/share/ansible_downloads/wine_access_2013_vorlage.tar.gz
+      remote_src: yes
+    register: wineprefix_tar_file
+
+  - name: Wineprefix benutzerabhaengig entpacken
+    unarchive:
+      src: /usr/local/share/ansible_downloads/wine_access_2013_vorlage.tar.gz
+      dest: /opt/
+      remote_src: yes
+      owner: "{{ item }}"
```

Comment:
Muss thomasn dort schreibend reinschreiben können? Sehen andere User die selben Daten? kann ein Virus die Datei MSACCESS.EXE modifizieren, wenn er als thomasn läuft? Wenn nur thomasn das nutzen soll, wäre es dann nicht in seinem /home/thomasn besser aufgehoben (mir ist gerade unklar, ob das Verzeichnis für den Testbetrieb ggf. überschrieben wird mit daten vom Babelfish ).